website/docs/API.md

```diff
@@ -30,6 +30,8 @@ title: API
     - [On a `ScrollView`](#on-a-scrollview)
     - [On a `FlatList`](#on-a-flatlist)
 - [`waitFor`](#waitfor)
+    - [`waitFor with jest fake timers`](#waitfor-with-jest-fake-timers)
```

Comment:
Can we simplify headers to avoid `waitFor` repetition, e.g. `Using with Jest fake timers`, `Modern fake timers`